### PR TITLE
tar: add hyphen for flags

### PR DESCRIPTION
**Pull request by Kjwon15**
Use modern options

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/tar.md [hunk 1]
```diff
@@ -5,32 +5,32 @@
...
 
 - Extract an archive in a target folder:
 
-`tar xf {{source.tar}} -C {{folder}}`
+`tar -xf {{source.tar}} -C {{folder}}`
 
 - Extract a gzipped archive in the current directory:
 
-`tar xzf {{source.tar.gz}}`
+`tar -xf {{source.tar.gz}}`
```
[the remark this answers is not in the document]
Comment (Kjwon15) [reply pages/common/tar.md]: tar don't need z option for untar.
That option is unnecessary and ignored

[the remark this answers is not in the document]
Comment (pxgamer) [reply pages/common/tar.md]: I thought it was necessary for Gzip. Which is what the example description mentions.

[the remark this answers is not in the document]
Comment (Kjwon15) [reply pages/common/tar.md]: Oh, Sorry. It was a fallback mode.

```
decompress-program
                     Controls  verbose  description of failures occurring when
                     trying to run alternative  decompressor  programs.   This
                     warning  is  disabled  by  default  (unless  --verbose is
                     used).  A common example of what you can get  when  using
                     this warning is:

                     $ tar --warning=decompress-program -x -f archive.Z
                     tar (child): cannot run compress: No such file or directory
                     tar (child): trying gzip

                     This  means  that tar first tried to decompress archive.Z
                     using compress, and, when that failed, switched to gzip.
```

[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/tar.md]: Hey, @pxgamer! I think one slipped through the net. Is there anything else that needs doing here to get this merged?

**Comment by slicer69** [comment]
Out of curiosity, why was the hyphen added to all the examples? The tar command doesn't need them and most people don't use the hyphen when running tar.

**Comment by sbrl** [comment]
@slicer69 It's because it's actually a short argument option - `tar` is odd in that it doesn't require it. For example, `tar caf file.txt` should technically be `tar -c -a -f file.txt`.

**Comment by slicer69** [comment]
Yes, I get that tar can do it either way, but why change the examples to a style that generally isn't used? You'll almost never see someone type "tar -c -a -f file.txt", it's almost always "tar caf file.txt". Wouldn't it make more sense to go with the way tar is typically invoked?

You say it "should technically be" the longer format, but according to whom? Even the man page uses both formats interchangeably.

**Pull request merged**